tpgtools/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GoogleCloudPlatform/magic-modules/tpgtools
 
-go 1.21
+go 1.22
```

Comment:
is 1.22 intended instead of 1.23?

Reply:
this was a mistake on my part. Must've not saved, thanks for catching this